hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -584,6 +585,39 @@ public static HoodieTableMetaClient initTableAndGetMetaClient(Configuration hado
     return metaClient;
   }
 
+  /**
+   * @param conf     file system configuration.
+   * @param basePath base path of the Hudi table.
+   * @return a new {@link HoodieTableMetaClient} instance.
+   */
+  public static HoodieTableMetaClient build(Configuration conf,
+                                            String basePath) {
```

Comment:
Don't think there is necessity to add three new builder methods, the original builder is more flexible to extend and there is no much gains to switch these new methods which also introduce bunden for maintainance.

Reply:
I moved these to the test utils as discussed offline.